packages/nouns-contracts/test/governance/NounsDAO/V2/dynamicQuorum.test.ts

```diff
@@ -27,96 +26,41 @@ describe('Dynamic Quorum', () => {
     gov = await deployGovernorV2(deployer);
   });
 
-  it('coefs all zeroes', async () => {
+  it('coefficient set to zero', async () => {
     const quorumVotes = await gov.dynamicQuorumVotes(12, 200, {
       minQuorumVotesBPS: 1000,
       maxQuorumVotesBPS: 4000,
-      quorumVotesBPSOffset: 500,
-      quorumLinearCoefficient: 0,
-      quorumQuadraticCoefficient: 0,
+      quorumCoefficient: 0,
     });
 
     expect(quorumVotes).to.equal(20);
   });
 
-  describe('Linear function', async () => {
-    it('stays flat before the offset value', async () => {
-      const quorumVotes = await gov.dynamicQuorumVotes(6, 200, {
-        minQuorumVotesBPS: 1000,
-        maxQuorumVotesBPS: 4000,
-        quorumVotesBPSOffset: 500,
-        quorumLinearCoefficient: parseUnits('1', 6),
-        quorumQuadraticCoefficient: 0,
-      });
-
-      expect(quorumVotes).to.equal(20);
+  it('increases linearly', async () => {
+    const quorumVotes = await gov.dynamicQuorumVotes(18, 200, {
+      minQuorumVotesBPS: 1000,
+      maxQuorumVotesBPS: 4000,
+      quorumCoefficient: parseUnits('1', 6),
```

Comment:
@eladmallel can you add one or two more tests with coefficients other 1?
e.g. 0.1 and 2

Reply:
https://github.com/nounsDAO/nouns-monorepo/pull/438/commits/bd4499a65215dc671704b494ca53ab51d3e4e0e4